test/FadeMixinSpec.js

```diff
@@ -13,6 +13,7 @@ describe('FadeMixin', function () {
         return (
           <div {...this.props} className='fade'>
             <span className='fade'/>
+            <textarea></textarea>
```

Comment:
Please do not add this here.
It belongs only to one test.
It would be best to place it exactly with that test case :cherries: